nucypher/characters/lawful.py

```diff
@@ -841,7 +831,7 @@ def __init__(self,
                            is_me=is_me,
                            checksum_address=checksum_address,
                            start_learning_now=False,  # Handled later in this function to avoid race condition
-                           federated_only=self._federated_only_instances,
+                           federated_only=self._federated_only_instances,  # TODO: 'Ursula' object has no attribute '_federated_only_instances' if an is_me Ursula is not inited prior to this moment
```

Comment:
To pick a nit: it needn't be an `is_me` *`Ursula`*, it can be *any* `Learner` who `is_me`.  (or at least, that was my intention; if that's not panning out it might be a bug)